batchflow/models/torch/base.py

```diff
@@ -789,6 +795,104 @@ def head(cls, inputs, **kwargs):
         """
         return cls.block(inputs, name='head', **kwargs)
 
+    # Model weights initialization
+    def init_weights(self, init_model_weights=None, init_zero_bias=False):
+        """ Initialize model weights with specific distribution and initialize biases to 0.
+
+        About weighs initialization, you can read here: `torch.nn.init <https://pytorch.org/docs/stable/nn.init.html>`_.
+
+        Parameters
+        ----------
+        init_model_weights : dict, True, or None
```

Comment:
Still believe that a callable with a set of pre-defined strings would be more convenient

Reply:
Ok